feat: add JSON language server support via vscode-json-languageserver (#1404)
oraios/serena: docs/01-about/020_programming-languages.md

* **Vue**    
  (3.x with TypeScript; requires Node.js v18+ and npm; supports .vue Single File Components with monorepo detection)
* **YAML**
* **JSON**  

>   (experimental; provides document symbol navigation and hover; must be explicitly specified as the main language; requires Node.js and npm)

Reviewer: that's not true, there is no concept of main language. It just should be added to the languages manually
```suggestion
  (experimental; must be explicitly added to the languages list; requires Node.js and npm)
```
